http/pom.xml

```diff
@@ -42,7 +42,7 @@
         <module>base</module>
         <module>bridge</module>
         <module>inventoryprinter</module>
-        <module>itest</module>
+       <!--<module>itest</module>-->
```

Comment:
Is this still required now `-DskipITs` is used?

Reply:
Yes, the regular unit tests also don't run with what's in master right now.